### ISSUE TITLE
Try to learn model on base speakers and then use model file to retrain on added one

### ISSUE THREAD

**Issue by papeye** [papeye/speaker-identification#123]
training takes long because we always train on every speaker. we could pre train model on base speakers and then retrain it on added ones but keeping validation set from the whole training set base + new speakers
